content/en/synthetics/guide/api_test_timing_variations.md

```diff
@@ -0,0 +1,119 @@
+---
+title: Understanding API test timings and troubleshooting their variations
+kind: documentation
+description: Understanding API test timings and troubleshooting their variations
+aliases:
+- /synthetics/api_test_timing_variations
+further_reading:
+- link: "https://docs.datadoghq.com/synthetics/metrics/#api-tests"
+  tag: "Documentation"
+  text: "Synthetics API test metrics"
+---
+
+
+## Overview
+
+Synthetics API tests collect several [**timing metrics**][1] which can be used to identify any bottlenecks in the communication between the client and your server. 
```

Comment:
Un-bolding the link because "timing metrics" are not UI elements.

```suggestion
You can identify bottlenecks in the communication between your server and the client with [timing metrics][1] collected by Synthetic API tests.

```